site/content/3.12/data-science/graph-analytics.md

```diff
@@ -0,0 +1,651 @@
+---
+title: Graph Analytics
+menuTitle: Graph Analytics
+weight: 123
+description: |
+  ArangoGraph offers Graph Analytics Engines to run graph algorithms on your
+  data separately from your ArangoDB deployments
+---
+Graph analytics is a branch of data science that deals with analyzing information
+networks known as graphs, and extracting information from the data relationships.
+It ranges from basic measures that characterize graphs, over PageRank, to complex
+algorithms. Common use cases include fraud detection, recommender systems,
+and network flow analysis.
+
+ArangoDB offers a feature for running algorithms on your graph data,
+called Graph Analytics Engines (GAEs). It is available on request for the
+[ArangoGraph Insights Platform](https://dashboard.arangodb.cloud/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic).
+Your ArangoDB deployment needs to use AWS as the provider.
+
+Key features:
+
+- **Separation of storage and compute**: GAEs are a solution that lets you run
+  graph analytics independent of your ArangoDB deployments on dedicated machines
+  optimized for compute tasks. This separation of OLAP and OLTP workloads avoids
+  affecting the performance of the transaction-oriented database systems.
+
+- **Fast data loading**: You can easily and efficiently import graph data from
+  ArangoDB and export results back to ArangoDB.
+
+- **In-memory processing**: All imported data is held and processed in the
+  main memory of the compute machines for very fast execution of graph algorithms
+  such as connected components, label propagation, and PageRank.
+
+## Workflow
+
+The following lists outlines how you can use Graph Analytics Engines (GAEs).
+How to perform the steps is detailed in the subsequent sections.
+
+0. Request beta access to the feature via __Request help__ in the
```

Comment:
One "beta" mention is still there, please remove it. May be it's even better (for more visibility) to put an admonition similar to the ML about enabling the feature.

Reply:
Already changed but missed to push